spring-cloud-stream-binder-rabbit-core/src/main/java/org/springframework/cloud/stream/binder/rabbit/properties/RabbitBinderConfigurationProperties.java

```diff
@@ -30,6 +31,8 @@
 
 	private int compressionLevel;
 
+	private String connectionNamePrefix;
```

Comment:
I think all these properties are configuration properties, therefore auto-completion assistant in the IDE should be supplied with some documentations on the matter.

I mean all configuration properties must have appropriate JavaDocs: https://docs.spring.io/spring-boot/docs/2.0.0.RELEASE/reference/htmlsingle/#configuration-metadata

Right, think might be addressed in the separate issue.